cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -430,6 +434,21 @@ void BuildStringDictionaryIndex(ColumnChunkDesc* chunks,
                                 int32_t num_chunks,
                                 rmm::cuda_stream_view stream);
 
+/**
+ * @brief Compute the amount of string data in DELTA_BYTE_ARRAY encoded pages.
+ *
+ * @param[in,out] pages All pages to compute lengths for
+ * @param[in] chunks Chunks that contain the pages
+ * @param[in] num_rows Total number of rows to read
+ * @param[in] min_row crop all rows below min_row
+ * @param[in] stream CUDA stream to use, default 0
+ */
+void ComputePageStringSizes(hostdevice_vector<PageInfo>& pages,
```

Comment:
Might be a good idea to throw a "Delta" somewhere in the name here to distinguish it a bit.